packages/bazel-bot/test-generate-googleapis-gen.sh

```diff
@@ -66,8 +66,8 @@ git -C googleapis-gen checkout -b other
 # Test!
 export GOOGLEAPIS_GEN=`realpath googleapis-gen-clone`
 export INSTALL_CREDENTIALS="echo 'Pretending to install credentials...''"
-export BUILD_TARGETS="//google/cloud/vision/v1:vision-v1-nodejs.tar.gz //google/bogus:api.tar.gz"
-export FETCH_TARGETS="//google/cloud/vision/v1:vision-v1-nodejs.tar.gz"
+export BUILD_TARGETS="//google/cloud/vision/v1:google-cloud-vision-v1-csharp //google/bogus:api"
```

Comment:
Looks like it replaces node with C#, which proves that C# works, but stops checking node, so there is a potential for regression. Can you please instead of replaceing node with C#, just add C#?

Reply:
adding a second working target does nothing to the test since it only tests that `a commit` is produced

Reply:
also the tests does not check for all languages, so I would argue that the potential of regression is unchanged by replacing one with another